base/doc/ltnews38.tex

```diff
@@ -149,6 +149,43 @@ \section{New or improved commands}
 
 \section{Code improvements}
 
+\subsection{Support for tabs in \cs{verb*}}
+
+\LaTeX{} converts tabs to spaces when typsetting. The same has been true to
```

Comment:
converts as TAB to a single space when typesetting.

(as tabs are often represented as several spaces it is not what what we mean if the plural form is used here.)

Reply:
I was trying to get at that multiple tabs might convert to multiple spaces, but TeX then collapses the result to one space. So perhaps 'converts a single tab to a single space, which is then treated like 'real' spaces in typesetting.'?

Reply:
something like this sounds good to me